Fix-Website/ApplicationInsights.config

```diff
@@ -0,0 +1,84 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Some of these files are fragments (I assume) from visual studio's basic website project config. Will clean up.